processor.py

```diff
@@ -21,6 +21,9 @@ def process(
         # print(documents)
         c = ""
         for i, j in documents.meta.items():
-            c += (f"{i} {j}. ")
+            if 'price' in i:
+                c += f"{i} {int(j) / 100}\n"
+            else:
+                c += f"{i} {j}\n"
```

Comment:
you could replace with
```
  documents.content = f"""Product named "{documents.content}", 
        for price "{documents.meta.get("price", 0)/100}",
        in currency "{documents.meta.get("currency")}",
        in country "{documents.meta.get("country_code")}", 
        is sold by "{documents.meta.get("merchant_name")}", 
        Described as "{documents.meta.get("descr")}",
        has EAN "{documents.meta.get("ean13")}",
        has the follow categories "{documents.meta.get("google_category_id", "").split('>')}"
        """
```
keep attention to
- missing fields in the DocumentStore search_fields
- carriage return (up to you if to remove, etc)

Reply:
yeah, going to merge @robertomena 's branch to mine to help with that.